pkg/addon/template.go

```diff
@@ -80,6 +81,11 @@ func NewTemplateData(
 		return nil, fmt.Errorf("failed to determine cloud provider name: %v", err)
 	}
 
+	// value is set by the mutating AdmissionHandler
+	if cluster.Spec.ClusterNetwork.IPVS.StrictArp == nil {
```

Comment:
I'd say that nil pointer should be defaulted to `true`. Otherwise existing clusters that didn't got modified just yet will start error-ing. And please use some central point (like resources package) to get this default value in both places, here and in the webhook.